common/sgx/eeid_verifier.c

```diff
@@ -393,7 +394,12 @@ static oe_result_t _eeid_free_claims_list(
 {
     OE_UNUSED(context);
     OE_UNUSED(claims_size);
-    free(claims);
+    for (size_t i = 0; i < claims_size; i++)
```

Comment:
Consider making `sgx_attestation_plugin_free_claims_list` part of include/openenclave/internal/sgx/plugin.h instead and using that here.

Reply:
That's a good suggestion; done. I kept the name as suggested in the design doc, but it's a bit long.

Reply:
Oh yeah, thanks for checking, I forgot that end of the story :-) Done now.